### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by dimondduck123** [iholston/lol-bot#118]
Can I make the bot play better in intermediate mode?

**Comment by iholston** [comment]
Yes, if you are using the latest release (v1.3.2) change the GAME_LOBBY_ID line in constants.py to 850. if you are using the gui you can do it on the config tab.

**Comment by dimondduck123** [comment]
okayyyyyyyyy i have 1.3.1 thx you are the best

**Comment by dimondduck123** [comment]
I downloaded the new version and it keeps getting stuck. Is there a solution?

**Comment by iholston** [comment]
which new version? and just drop the latest log in the logs folder in here.
